src/routing/pages/probe/GleanDetails.svelte

```diff
@@ -267,9 +277,9 @@
         Export to JSON
       </button>
       <LookerLink
-        product="fenix"
+        product="glean"
         variants={$store.probe.variants}
-        sendInPings={$store.probe.send_in_pings}
+        sendInPings={$store.productDimensions.ping_type}
```

Comment:
FOG page LookerLink area is greyed out:

<img width="234" alt="CleanShot 2022-04-14 at 15 17 27@2x" src="https://user-images.githubusercontent.com/28797553/163460903-af73625e-afbf-4fcb-984b-137e9d5af7fd.png">

which indicates that there's no Looker available for FOG probes, which is not true. Because FOG now have different ping types, we need to change how we get the ping information for LookerLink component.

<img width="231" alt="CleanShot 2022-04-14 at 15 13 23@2x" src="https://user-images.githubusercontent.com/28797553/163460856-ee8eb8fe-c83f-4fd7-87c6-435a035c3473.png">